pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -88,22 +88,18 @@ class ModificationsExportDialog extends React.Component {
                    title={_("Automation script") }>
                 <Tabs activeKey={this.state.active_tab} onSelect={this.handleSelect}>
                     <Tab eventKey="shell" title={_("Shell script")}>
-                        <pre>
-                            {this.props.shell.trim()}
-                        </pre>
+                        <TextArea resizeOrientation='vertical' isReadOnly defaultValue={this.props.shell.trim()} />
                     </Tab>
                     <Tab eventKey="ansible" title={_("Ansible")}>
-                        <pre>
-                            {this.props.ansible.trim()}
-                        </pre>
-                        <div>
-                            <span className="fa fa-question-circle fa-xs" />
+                        <TextArea resizeOrientation='vertical' isReadOnly defaultValue={this.props.ansible.trim()} />
+                        <div className="ansible-docs-link">
+                            <QuestionCircleIcon size="xs" />
```

Comment:
This black icon is just too punchy.
I would use `<OutlinedQuestionCircleIcon />` (without xs).

Before:
![weirdicons](https://user-images.githubusercontent.com/12330670/110683111-fac57e80-81db-11eb-903d-18cac7338076.png)

With icons I suggested:
![newicons](https://user-images.githubusercontent.com/12330670/110683184-0b75f480-81dc-11eb-9189-be704319d00b.png)